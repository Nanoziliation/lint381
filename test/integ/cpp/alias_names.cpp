using Foo = int;

using bar_t = int;
