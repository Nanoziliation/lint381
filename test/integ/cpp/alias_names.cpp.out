alias_names.cpp:1:7: error: Alias 'Foo' should end with '_t'
using Foo = int;
      ^^^
alias_names.cpp:3:7: error: Alias 'bar_t' should be capitalized
using bar_t = int;
      ^^^^^
